solutions/standard/variables.tf

```diff
@@ -0,0 +1,141 @@
+# Input variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key"
+  sensitive   = true
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
+  default     = []
+}
+
+variable "prefix" {
+  type        = string
+  description = "Prefix to append to all resources created by this deployable architecture"
+  default     = "ease-da"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "The name of an existing resource group to provision resources in to. If not set a new resource group will be created using the prefix variable value"
+  default     = null
+}
+
+variable "plan" {
+  type        = string
+  description = "The desired pricing plan for IBM Enterprise Application Service instance."
+  default     = "free"
+  validation {
+    condition     = contains(["trial", "standard", "free", "staging"], var.plan)
+    error_message = "The only values accepted for the plan field are free, standard, staging and trial."
+  }
+}
+
+variable "region" {
+  type        = string
+  description = "The desired region for deploying IBM Enterprise Application Service instance."
+  default     = "us-east"
+}
+
+variable "source_repo" {
+  type        = string
+  description = "The URL for the repository storing the source code of the application to deploy through IBM Cloud Enterprise Application Service."
+  default     = null
+}
+
+variable "config_repo" {
+  type        = string
+  description = "The URL for the repository storing the configuration to use for the application to deploy through IBM Cloud Enterprise Application Service."
+  default     = null
+}
+
+# github token configuration - the token or the tuple secrets manager id, secrets manager region and secret id are alternative
+# if both are present the tuple for secrets manager has higher priority
+
+variable "repos_git_token_existing_secrets_manager_id" {
+  type        = string
+  description = "The existing Secrets Manager instance to retrieve the GitHub token value. If not null, var.repos_git_token value will be ignored."
+  default     = null
+}
+
+variable "repos_git_token_existing_secrets_manager_region" {
+  type        = string
+  description = "The existing Secrets Manager instance region to retrieve the GitHub token value."
+  default     = "us-south"
+}
+
+variable "repos_git_token_secret_id" {
+  type        = string
+  description = "The secretID where the value for the GitHub token is stored in the existing Secrets Manager instance."
+  default     = null
+}
+
+variable "repos_git_token" {
+  type        = string
+  description = "The GitHub token to read from the application and configuration repos. If var.repos_git_token_existing_secrets_manager_id is not null, var.repos_git_token is not used."
```

Comment:
don't use "var." in DA variable descriptions. Doesn't make sense in projects land :)
Instead do something like "If no value is passed for the `repos_git_token_existing_secrets_manager_id` input ...."